### PR TITLE
[master] Jenkins pipeline - NoSQL (MongoDB) test execution fix

### PR DESCRIPTION
**Pull request by rfelcman**
This fix reflects latest changes in JPA tests.

Signed-off-by: Radek Felcman <radek.felcman@oracle.com>

### CODE REVIEW

File: etc/jenkins/pr_verify.groovy [hunk 1]
```diff
@@ -159,9 +159,7 @@ spec:
                     steps {
                         container('el-build') {
                             sh """
-                                /opt/bin/mongo-start.sh
-                                mvn -B -V verify -pl :org.eclipse.persistence.nosql -P staging,mongodb
-                                /opt/bin/mongo-stop.sh                                
+                                mvn -B -V verify -pl :org.eclipse.persistence.jpa.testapps.nosql.mongo -P staging
```
Comment (lukasj) [inline etc/jenkins/pr_verify.groovy:162]: the step can be completely removed, tests run within regular build already

Comment (rfelcman) [reply etc/jenkins/pr_verify.groovy]: OK mentioned `stage` was removed.

**Comment by rfelcman** [comment]
JPA LRG stage in `pr_verify.groovy` was changed too to call all JPA tests

```
mvn -B -V verify -pl :org.eclipse.persistence.jpa.test -P staging,mysql
mvn -B -V verify -rf :org.eclipse.persistence.jpa.testapps -P staging,mysql
```

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**